biomage/stage/stage.py

```diff
@@ -104,6 +105,52 @@ def get_latest_master_sha(chart, token):
     raise Exception("Invalid repository supplied.")
 
 
+def get_sandbox_id(templates, manifests):
+    # Generate a sandbox name and ask the user what they want theirs to be called.
+    sandbox_id = hashlib.md5(manifests.encode()).digest()
+    sandbox_id = anybase32.encode(sandbox_id, anybase32.ZBASE32).decode()
+    fragments = (
+        os.getenv("USER", ""),
```

Comment:
Can you add another non-system environment variable (e.g. `BIOMAGE_NICK`) that we can set freely and put it on a higher priority? I'd like to customize my name, but I don't want to change `USER` as it will mess up a lot of configs XD